ng-client/package.json

```diff
@@ -28,7 +28,7 @@
     "hammerjs": "2.0.8",
     "moment": "2.17.1",
     "rxjs": "5.1.0",
-    "zone.js": "0.8.4"
+    "zone.js": "0.8.5"
```

Comment:
This is about all that I can give you. https://github.com/angular/angular/issues/15185 I believe it has to do with their approach to overriding the global Error object - https://github.com/angular/zone.js/blob/v0.8.5/CHANGELOG.md